commands/customObject/schema/update.ts

```diff
@@ -65,7 +65,7 @@ export async function handler(
     name =
       providedName && typeof providedName === 'string'
         ? providedName
-        : await listPrompt(i18n(`commands.customObject.subcommands.schema.subcommands.update.selectSchema`), {
+        : await listPrompt<string>(i18n(`commands.customObject.subcommands.schema.subcommands.update.selectSchema`), {
```

Comment:
This one particular call got a little upset with the generic and was requiring an explicit `<string>` 🤷